src/test/scala/firrtlTests/LoweringCompilersSpec.scala

```diff
@@ -120,6 +120,8 @@ class LoweringCompilersSpec extends FlatSpec with Matchers {
 
     val patched = scala.collection.immutable.TreeMap(m.toArray:_*).values.flatten
 
+    println(patched.map(_.getName).mkString("\n"))
```

Comment:
I presume this println will get removed after the tests pass?

Reply:
Good find! This was never supposed to be committed!